docs/overrides/main.html

```diff
@@ -1,5 +1,10 @@
 {% extends "base.html" %}
 
+{% block announce %}
+👋 Powertools for Python v1 will no longer receive updates or releases after 31/03/2023!
+We encourage you to read important information's <a href="/upgrade/#end-of-support-v1">here</a>.
```

Comment:
```suggestion
We encourage you to read our <a href="/upgrade/#end-of-support-v1">upgrade guide on how to migrate to v2</a>.
```

It's great, but it can be even better by being explicit: "We encourage you to do <X>". If they read the upgrade guideline or not, that's a secondary goal :)